packages/main/src/plugin/container-registry.ts

```diff
@@ -1070,7 +1072,7 @@ export class ContainerProviderRegistry {
 
   getImageName(inspectInfo: Dockerode.ImageInspectInfo): string {
     const tags = inspectInfo.RepoTags;
-    if (!tags) {
+    if (!tags?.[0]) {
```

Comment:
```suggestion
    if (!tags || tags.length === 0 || !tags[0]) {
```

Reply:
hi @axel7083 , could you give me why we would need these 3 condition rather than one ?

Reply:
I think when editing at some point I received an error telling me to merge the condition as well

Reply:
![image](https://github.com/user-attachments/assets/4411e2cc-5e75-4b18-8da2-fe0c86a9c391)

I just checked, and there is no need, as javascript do not raise error while accessing index that does not exists !

Reply:
yes the optional chaining is checking that for us